README.md

```diff
@@ -41,8 +41,8 @@ brew install jaxxstorm/tap/stunner
 Download the binary from releases
 
 
-```
-VERSION=v0.0.8
+```bash
+VERSION=v0.0.10
 curl -L "https://github.com/jaxxstorm/stunner/releases/download/${VERSION}/stunner-${VERSION}-linux-amd64.tar.gz" | tar -xz
```

Comment:
If the assets didn't have the version in the file name this could be simplified to latest via:

`https://github.com/jaxxstorm/stunner/releases/latest/download/stunner-linuxamd64.tar.gz`